.github/workflows/lint.yml

```diff
@@ -1,13 +1,64 @@
-name: Lint 
-on: [push, pull_request]
+name: Lint
+on: [push]
 jobs:
   lint:
     name: Lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: actions/setup-node@v1
         with:
-          node-version: '10.x'
+          node-version: "18"
       - run: npm install
       - run: npm run lint
+
+  compare-output:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18"
+      - run: npm install
+      - run: npx swagger-cli bundle -t json -w 300 main.yaml > doc/compiled_new.json
+      - run: diff doc/compiled.json doc/compiled_new.json
+
+  compile-go:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.6
+      - name: Build
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          npm install
+          npm run generate.go
+          cd clients/go && go install ./...
```

Comment:
any reason for using `install` over `build`?

Reply:
`go build` just works as well here and changed it. `go install` is also copying the output to `$GOPATH/bin` and maybe it doesn't even do anything when there is no main function like in  this library